### PR TITLE
[feg] Integrate base accounting reporting into aaa service & radius

### PR DESCRIPTION
**Pull request by emakeev**
Signed-off-by: Evgeniy Makeev <evgeniym@fb.com>

## Summary

Integrate base accounting reporting into aaa service & radius. 
Adds optional ability to AAA service to call base_acct service to report service data usage.

## Test Plan

run unit tests; feg make precommit; radius gen, build, test

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: feg/protos/mconfig/mconfigs.proto [hunk 1]
```diff
@@ -162,6 +162,7 @@ message AAAConfig {
     // enable event logging for aaa events
     bool EventLoggingEnabled = 5;
     RadiusConfig RadiusConfig = 6;
+    bool AcctReportingEnabled = 7; // enable accounting reporting to the module's orc8r service
```
Comment (themarwhal) [inline feg/protos/mconfig/mconfigs.proto]: minor nit but I think if you put the comment in the line above, it gets included in the generated files.

**Review by themarwhal — APPROVED** [review]

File: feg/protos/mconfig/mconfigs.proto (hunk 1 above)
Comment (emakeev) [reply feg/protos/mconfig/mconfigs.proto]: 👍

File: feg/gateway/utils/configs.go [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+)
+
+// GetValueUint64 returns value of the flagValue if it exists, or defaultValue if not
+func GetValueUint64(flagName string, defaultValue uint64) uint64 {
+	if len(flagName) > 0 {
+		value, err := getUint64FlagValue(flagName)
+		if err != nil {
+			return value
+		}
+
```
Comment (uri200) [inline feg/gateway/utils/configs.go]: maybe this space shouldnt be there?

**Review by uri200 — APPROVED** [review]

File: feg/gateway/utils/configs.go (hunk 2 above)
Comment (emakeev) [reply feg/gateway/utils/configs.go]: 👍

**Pull request merged**